docs/source/cn/math_ops.py

```diff
@@ -1491,3 +1490,32 @@
         
     """,
 )
+
+reset_docstr(
+    oneflow.std,
+    r"""
+    返回 :attr:`input` 张量在 :attr:`dim` 维度每行的标准差。如果 :attr:`dim` 是一个维度列表，则减少 (reduce) 所有的维度。
+
+    如果 keepdim 为真，输出张量与输入张量大小相同，除了在 :attr:`dim` 维度的大小变为1。否则， :attr:`dim` 将被压缩，导致输出张量拥有 1 (或者 len(dim)) 个更少的维度。
+
+    如果 :attr:`unbiased` 为 ``False`` ，则标准差将通过有差估算器计算。否则，贝塞尔校正将被使用。
+
+    参数：
+        input (Tensor): 输入张量
```

Comment:
```suggestion
       - **input** (Tensor): 输入张量
```

这里的风格好像和其它地方不一样？请同步检查其它地方

Reply:
已解决 已检查